### ISSUE TITLE
Question about the validation set

### ISSUE THREAD

**Issue by alberthli** [jishengpeng/WavTokenizer#62]
Hi - really great work on WavTokenizer! I was trying to reproduce some of the results from the paper on the small models, but could not find any information on the val split on LibriTTS. Was the validation set just generated as a proportion of the training set, or was it something like the `test-clean` split from the originally released data?

Thanks!

**Comment by jishengpeng** [comment]
> Hi - really great work on WavTokenizer! I was trying to reproduce some of the results from the paper on the small models, but could not find any information on the val split on LibriTTS. Was the validation set just generated as a proportion of the training set, or was it something like the `test-clean` split from the originally released data?
> 
> Thanks!

Thank you very much for your attention. The LibriTTS dataset consists of a training set, validation set, and test set. In our work, we used the validation set of LibriTTS as the validation set for WavTokenizer.

**Comment by alberthli** [comment]
Hi @jishengpeng, this doesn't clear things up for me, so maybe I'm misunderstanding. LibriTTS has the following [official](https://pytorch.org/audio/main/generated/torchaudio.datasets.LIBRITTS.html) splits from `torchaudio`:
* `train-clean-100`
* `train-clean-360`
* `train-other-500`
* `test-clean`
* `test-other`
* `dev-clean`
* `dev-other`
It does not seem like there is an official validation set amongst these, which led to my question. Please let me know if the dataset from your work does not follow these file splits, and if not, where you found the val split. Thank you!

**Comment by jishengpeng** [comment]
> Hi @jishengpeng, this doesn't clear things up for me, so maybe I'm misunderstanding. LibriTTS has the following [official](https://pytorch.org/audio/main/generated/torchaudio.datasets.LIBRITTS.html) splits from `torchaudio`:
> 
> * `train-clean-100`
> * `train-clean-360`
> * `train-other-500`
> * `test-clean`
> * `test-other`
> * `dev-clean`
> * `dev-other`
> 
> It does not seem like there is an official validation set amongst these, which led to my question. Please let me know if the dataset from your work does not follow these file splits, and if not, where you found the val split. Thank you!

We use the dev-clean and dev-other as the validation set in WavTokenizer.

**Comment by alberthli** [comment]
Thank you!

**Comment by jishengpeng** [comment]
> Follow-up question: do you train the model on all of the 3 training splits above, or only on a subset of them?

all of 3 training splits above.
